GoogleUtilities/Tests/Unit/Environment/GULKeychainStorageTests.m

```diff
@@ -18,6 +18,9 @@
 #if !TARGET_OS_MACCATALYST
 // Skip keychain tests on Catalyst.
 
+#if !SWIFT_PACKAGE
+// TODO: Investigate why keychain tests fail on iOS with Swift Package Manager.
```

Comment:
I guess it is because a host app is needed for keychain access and it looks like SPM tests don't have one.